src/docs/product/discover-queries/index.mdx

```diff
@@ -82,11 +84,23 @@ Share your queries as often as you want. You can share URLs with other users who
 
 ### Delete Queries
 
-On the **Discover** page, each saved query card has an ellipsis that will open a context menu. From here, you can delete the query. This action is irreversible. You can also delete the query within the Query Results view by clicking the trash can in the upper right.
+On the **Discover** page, each saved query card has an ellipsis that will open a context menu. From here, you can delete the query. This action is irreversible. You can also delete the query within the [query results](#query-results) view by clicking the "Delete Saved Query" option in the context menu.
 
 ### Add to Dashboard
 
-Queries can also be added to <SandboxLink scenario="dashboards" projectSlug="react">custom dashboards</SandboxLink> as widgets. You can find the "Add to Dashboard" button by opening the context menu in the **Discover** page or within the [query results](#query-results) view. When you click "Add to Dashboard", a form opens where you can name your widget, select your target dashboard, and make other changes before saving your new widget.
+Queries can also be added to <SandboxLink scenario="dashboards" projectSlug="react">custom dashboards</SandboxLink> as widgets. You can find the "Add to Dashboard" button by opening the context menu in the **Discover** page or within the context menu of the [query results](#query-results) view. When you click "Add to Dashboard", a form opens where you can name your widget, select your target dashboard, and make other changes before saving your new widget.
+
+### Set As Default
+
+<Note>
+
+The **Discover** homepage is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+Queries can be used as your **Discover** homepage. You can find the "Set As Default" button by opening the context menu in the **Discover** page or within the [query results](#query-results). You can customize your homepage by configuring how your columns and charts appear, and then clicking "Set As Default". Navigating to **Discover** through the left sidebar will now open a new query builder with this configuration pre-loaded.
```

Comment:
Since these two features (seeing the query builder as the homepage + being able to set a homepage) are part of the same feature, I'm rolling this all into one bit of content here. When this goes into GA, then it'll make sense to split it back up the way you had it before, with some tweaks.

```suggestion
If your organization is part of the Early Adopter program, when you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query. You can customize your homepage by configuring how your columns and charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded.

If you don't want to use this configuration as your default homepage anymore, click "Remove Default"; the homepage will revert to the "All Events" query.

To navigate to the query management page, where you can find pre-built and saved queries, click the "Saved Queries" button.

```